### PR TITLE
Manually spell checked the file.

### PR DESCRIPTION
**Pull request by ceddc**
Fixed some typo

Please review before accepting pull request

### CODE REVIEW

File: ETC_en.tsv [hunk 1]
```diff
@@ -10114,12 +10114,12 @@ ETC_20150323_010113	Acquired kindling! {nl}Put it in the fire for the Tree Root
...
 ETC_20150323_010115	Flame is blazing up!
 ETC_20150323_010116	Successfully burned Tree Root Crystal.
-ETC_20150323_010117	Trapped the evil aura in the box.
+ETC_20150323_010117	trapped the evil aura in the box.
 ETC_20150323_010118	The evil aura is being absorbed into the box.
 ETC_20150323_010119	It will automatically explode when you take the core of Tree Root Crystal from the monster!
 ETC_20150323_010120	Tree Root Crystal Reacting
 ETC_20150323_010121	Soul of Pilgrim entrapped by the monster is released!{nl}Destroy the entrapped Soul of Pilgrim to release him!
-ETC_20150323_010122	Trapped Soul of Pilgrim
+ETC_20150323_010122	trapped Soul of Pilgrim
```
Comment (Sunaries) [inline ETC_en.tsv]: Why is this uncapitalized? It's either an NPC or monster.

File: ITEM_en.tsv [hunk 2]
```diff
@@ -3229,16 +3229,16 @@ ITEM_20150317_003388	A great herb requested by Master Cleric
 ITEM_20150317_003389	Goddess Offering Pocket
 ITEM_20150317_003390	An offering pocket that the cleric uses to warship Goddess statue
 ITEM_20150317_003391	Balm
-ITEM_20150317_003392	A balm uses to repair Goddess staute. The feature is unusual and soft scent
+ITEM_20150317_003392	A balm uses to repair Goddess statue. The feature is unusual and soft scent
```
Comment (Sunaries) [inline ITEM_en.tsv:3232]: Goddess statue -> Goddess Statue

File: ITEM_en.tsv [hunk 3]
```diff
@@ -3531,22 +3531,22 @@ ITEM_20150317_003736	Complete scriptures
...
-ITEM_20150317_003750	Branches of bee tree contains the power of Goddness Austeza
+ITEM_20150317_003750	Branches of bee tree contains the power of Goddess Austeza
 ITEM_20150317_003751	Ashes of burned demon
-ITEM_20150317_003752	Ashes of devil burned due to Goddness Austeza's power
+ITEM_20150317_003752	Ashes of devil burned due to Goddess Austeza's power
 ITEM_20150317_003753	Beads of residue
 ITEM_20150317_003756	Materials to make scarecrow
 ITEM_20150317_003761	The third revelation found from Thorn Forest
-ITEM_20150317_003762	The fourth revelation found from Royaltobm
+ITEM_20150317_003762	The fourth revelation found from Royaltomb
```
Comment (Sunaries) [inline ITEM_en.tsv:3549]: Royaltomb > Royal Tomb

File: ITEM_en.tsv [hunk 4]
```diff
@@ -4678,33 +4678,33 @@ ITEM_20150428_005064	The second revelations discovered in the Tenet Chapel. It i
...
 ITEM_20150428_005071	Reagents of testing soul that Graschius passed.
 ITEM_20150428_005072	Reagents for testing
-ITEM_20150428_005073	It is said to distingush the monster which killed the pilgrims to the curse of anger to spary the monster. However, this is still a test reagent.
+ITEM_20150428_005073	It is said to distinguish the monster which killed the pilgrims to the curse of anger to spary the monster. However, this is still a test reagent.
 ITEM_20150428_005074	The fifth revelation found in the Cathedral. It is able to read.
 ITEM_20150511_005075	Featherfoot Costume
 ITEM_20150511_005076	Lama Costume
 ITEM_20150511_005077	Crystal for resonance
-ITEM_20150511_005078	Myterious crystal dropped by Vubbe. It can be resonating any device at the second floor of 2 block.
+ITEM_20150511_005078	Mysterious crystal dropped by Vubbe. It can be resonating any device at the second floor of 2 block.
```
Comment (Sunaries) [inline ITEM_en.tsv]: _A_ mysterious crystal....

File: ETC_en.tsv (hunk 1 above)
Comment (ceddc) [inline ETC_en.tsv]: A mistake on using replace function. Will fix

File: ITEM_en.tsv (hunk 2 above)
Comment (ceddc) [inline ITEM_en.tsv:3232]: why so? a statue is an object, even for a Goddess no?

File: ITEM_en.tsv (hunk 3 above)
Comment (ceddc) [inline ITEM_en.tsv:3549]: I just fixed typo. There is 32 "Royal Tomb" in files and  12 "RoyalTomb" didn't check if Korean version is the same.

File: ITEM_en.tsv (hunk 4 above)
Comment (ceddc) [inline ITEM_en.tsv]: Just fixed typo. But will fix.

File: ETC_en.tsv [hunk 5]
```diff
@@ -460,7 +460,7 @@ ETC_20150317_000459	Strictly speaking, it is a creature of magic whose name wish
 ETC_20150317_000460	Echad
 ETC_20150317_000461	Is a creature of magic that was created to protect the king's tomb. However, it is the stronger the Asmodian also been rice goddess of power.
 ETC_20150317_000462	Shtayim
-ETC_20150317_000463	Despite it's appearance, Creatures of this magic has recieved the blessing of the goddess. Can be fatal to Asmodian.
+ETC_20150317_000463	Despite it's appearance, Creatures of this magic has received the blessing of the goddess. Can be fatal to Asmodian.
```
Comment (Sunaries) [inline ETC_en.tsv:463]: blessing of the goddess -> Blessing of the Goddess
Asmodian -> demons

File: ITEM_en.tsv (hunk 2 above)
Comment (Sunaries) [inline ITEM_en.tsv:3232]: The name is not separate and therefore it is Goddess Statue (여신상). It's one object.

File: ETC_en.tsv (hunk 5 above)
Comment (ceddc) [inline ETC_en.tsv:463]: I just fixed the typo.
Will need to check consistency in all files in a future pull about other stuff :-)

Thanks btw :+1:

File: ITEM_en.tsv (hunk 2 above)
Comment (ceddc) [inline ITEM_en.tsv:3232]: ok.
This pull is just for fixing some typo.
There is 6 other line where Goddess Statue don't match case.
Will be fixed in another pull.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**